SystemInfo/SystemInfoCollectorRegistry.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * File containing the SystemInfoCollectorRegistry interface.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo;
+
+/**
+ * A registry of SystemInfoCollectors.
+ */
+interface SystemInfoCollectorRegistry
+{
+    /**
+     * Adds SystemInfoCollectors, each with an identifier string, to the registry.
+     *
+     * @param \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector[] $items A hash of SystemInfoCollectors, with identifier string as key.
+     */
+    public function addItems(array $items);
```

Comment:
You commented on "registry" in the name of the getter, not for the setter. I removed it from both, to keep them consistent. Could be what you meant, of course :)